Source/Core/DolphinQt/Config/FreeLookWidget.cpp

```diff
@@ -35,6 +36,17 @@ void FreeLookWidget::CreateLayout()
   m_enable_freelook->SetDescription(
       tr("Allows manipulation of the in-game camera.<br><br><dolphin_emphasis>If unsure, "
          "leave this unchecked.</dolphin_emphasis>"));
+#ifdef USE_RETRO_ACHIEVEMENTS
+  bool hardcore = Config::Get(Config::RA_HARDCORE_ENABLED);
+  m_enable_freelook->setEnabled(!hardcore);
```

Comment:
I think if we used the hardcorewarning with all the controls disabled, that'd be a better experience that forcing the freelook value which means if the user disabled hardcode mode they'd have to remember to re-enable this.  You would have to also disable the freelook controller logic though (probably need to do that anyway?)

Reply:
Not entirely certain I follow what you're asking for here. What do you expect to see when hardcore mode is enabled?